server/config/constants.js

```diff
@@ -35,3 +35,5 @@ export const EVENT = {
 export const AUTH_ERROR_CODE = {
     INVALID_CLIENT: 'invalid_client'
 };
+
+export const TIMEOUT_ERROR_MESSAGE = `Timed out after`;
```

Comment:
Nit: let's use single quotes

```suggestion
export const TIMEOUT_ERROR_MESSAGE = 'Timed out after';
```

Reply:
pushed!